packages/fluentui/react-builder/src/components/CanvasWindow.tsx

```diff
@@ -33,15 +33,13 @@ export type CanvasWindowProps = {
   onSourceCodeError: (code: any, error: any) => void;
   onSelectComponent: (jsonTreeElement: any) => void;
   onSwitchToStore: () => void;
+  onLoadFromUrl: () => void;
 };
 
 export const CanvasWindow: React.FunctionComponent<CanvasWindowProps> = (props: CanvasWindowProps) => {
   const HEADER_HEIGHT = '3rem';
   const [headerMessage, setHeaderMessage] = React.useState('');
-  const codeSandboxData = getCodeSandboxInfo(
-    props.state.jsonTree,
-    renderElementToJSX(renderJSONTreeToJSXElement(props.state.jsonTree)),
```

Comment:
for reasons below my comprehension, this call started parsing React tree outside of the canvas content after state was replaced - all the way to the docs site.

Debugging it was fun but it was not successful, so the next best option to fixing the problem turned out to be  avoiding it. codesandbox loads the code from the state which according to my tests works